### ISSUE TITLE
Not working with sveltekit 2.x?

### ISSUE THREAD

**Issue by rchrdnsh** [j4w8n/supakit#18]
tried to npm install supakit in a kit 2.4.1 app and got the following error during instalation:

```terminal
npm install supakit
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: rykr-kit@0.0.1
npm ERR! Found: @sveltejs/kit@2.4.1
npm ERR! node_modules/@sveltejs/kit
npm ERR!   dev @sveltejs/kit@"^2.4.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @sveltejs/kit@"^1.20.4" from supakit@1.5.0
npm ERR! node_modules/supakit
npm ERR!   supakit@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/rchrdnsh/.npm/_logs/2024-01-19T21_08_39_568Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/rchrdnsh/.npm/_logs/2024-01-19T21_08_39_568Z-debug-0.log
```

**Comment by j4w8n** [comment]
Yeah I haven't gotten around to making it compat with v2 yet.

**Comment by rchrdnsh** [comment]
hmmmmmm...actually...mind taking a look at this discord issue?

The code you gave me a while back to use supabase auth in a svelte store in a svelte context is causing a deadlock now...

I don't know how to fix it...

https://discord.com/channels/457912077277855764/1198010610210852885

**Comment by j4w8n** [comment]
I've now released v2.0.1, which requires the following:

- @sveltejs/kit ^2.4.1
- @supabase/supabase-js ^2.39.3
- svelte >=4.2.9 (I've not tested it with v5, but figured I'd leave the possibility open)
- vite ^5.0.12

**Comment by rchrdnsh** [comment]
yup, installed perfectly fine now, thank yooooooo :-)
